### ISSUE TITLE
Job Detail: loader out of place

### ISSUE THREAD

**Issue by ondrejhlavacek** [keboola/kbc-ui#2054]
je tam trochu jako pěst na oko

<img width="1440" alt="screen shot 2018-09-18 at 16 06 38" src="https://user-images.githubusercontent.com/497675/45693249-f36a1080-bb5c-11e8-9a43-1f5dd118ae82.png">

**Comment by janmichek** [comment]
Vidim 3 ruzny signalizace loadingu. Nejsem spokojenej ani s jednim .]

jinak je to dost obecny problem https://github.com/keboola/indigo-ui/issues/186

**Comment by ondrejhlavacek** [comment]
Jo, myslel jsem ten spodní, kterej mě jen irituje, jak je nezarovnanej. Se zbytkem souhlas :-)
